_tools/check_licence.ts

```diff
@@ -31,10 +32,17 @@ for await (
   })
 ) {
   const content = await Deno.readTextFile(path);
+
   if (!content.includes(COPYRIGHT)) {
-    console.error(`Missing/incorrect copyright header: ${path}`);
-    if (!failed) {
+    if (Deno.args.includes("--check")) {
+      console.error(`Missing/incorrect copyright header: ${path}`);
       failed = true;
+    } else {
+      const newline = isWindows ? "\r\n" : "\n";
```

Comment:
I'm not sure about windows situation of this, but when a windows user pushes the commits to github, these `\r\n`s are changed to `\n`?

Reply:
Would it help to add a `.gitattributes` file with `eol` (https://git-scm.com/docs/gitattributes#_eol)?

Reply:
Deno CLI repo seems having that setting: `* text=auto eol=lf`

https://github.com/denoland/deno/blob/9daf6e197a642a88d79614fb53888b5bb954463a/.gitattributes#L2

Reply:
and we already have a similar setting in .editorconfig.

https://github.com/denoland/deno_std/blob/eca4dbfab0a8f5ab5e80daa255c4024e76c042ce/.editorconfig

Reply:
> Deno CLI repo seems having that setting: `* text=auto eol=lf`

So, add it here too?

Reply:
Ah, deno_std also already have it. https://github.com/denoland/deno_std/blob/6bd933286079d9afbb588092e963e0565c230dfb/.gitattributes#L2

I think we don't need to use "\r\n" on windows. Let's always use "\n" here